File: platforms/swift/examples/geometry/mesh/set_instance_count.swift
import FragmentColor
let m = Mesh()
try m.addVertices([
    [-0.01, -0.01],
    [ 0.01, -0.01],
    [ 0.00,  0.01],
])
// Draw one million instances, fetching per-particle data from a storage buffer.
m.setInstanceCount(1_000_000)

File: platforms/web/examples/geometry/mesh/set_instance_count.js
import { Mesh } from "fragmentcolor";
const m = new Mesh();
m.addVertices([ [-0.01, -0.01], [ 0.01, -0.01], [ 0.00,  0.01], ]);
// Draw one million instances, fetching per-particle data from a storage buffer.
m.setInstanceCount(1_000_000);